src/subdomains/generic/user/services/webhook/dto/webhook.dto.ts

```diff
@@ -4,10 +4,14 @@ export enum WebhookType {
   PAYMENT = 'Payment',
   KYC_CHANGED = 'KycChanged',
   KYC_FAILED = 'KycFailed',
+  ACCOUNT_MERGE = 'AccountMerge',
 }
```

Comment:
Hm yeah don't know if thats not a little bit confusing, have to think about it 🤔  because if we change anything else on account information that would also be `AccountChanged` and "merging" in this case is something different I would say to "changing"

Reply:
Hm yes 🤔 I was thinking from the "slave" perspective. As a slave, the only change you see is a new account id 😄